test_haystack/solr_tests/test_management_commands.py

```diff
@@ -3,20 +3,26 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 import datetime
+import pysolr
+import os
 from tempfile import mkdtemp
+try:
+    from StringIO import StringIO
```

Comment:
Since we don't support Python 2.6 any more we can make this `from io import StringIO` on everything since Python 2.7 supports that.